frontend/apps/core-app/src/screens/RecipientListScreen/RecipientListScreen.container.tsx

```diff
@@ -1,22 +1,29 @@
 import * as React from 'react';
-import { NavigationProp, useNavigation } from '@react-navigation/native';
-import { Button } from 'native-base';
+import { StackScreenProps } from '@react-navigation/stack';
 
-import { RootParamList } from '../../navigation/types';
 import { RecipientListTableContext } from '../../components/RecipientListTable/RecipientListTableContext';
 import {
   RecipientListTableEntry,
   SortedFilteredTable,
 } from '../../components/RecipientListTable/types';
+import { RecipientNavigatorParamList } from '../../navigation/recipients';
+import { routes } from '../../constants/routes';
+import { useRecipientForms } from '../../contexts/RecipientForms';
 
 import { RecipientListScreenComponent } from './RecipientListScreen.component';
 
-export const RecipientListScreenContainer: React.FC = () => {
-  const navigation = useNavigation<NavigationProp<RootParamList>>();
+type Props = StackScreenProps<RecipientNavigatorParamList, 'recipientsList'>;
+
+export const RecipientListScreenContainer: React.FC<Props> = ({
+  navigation,
+}) => {
+  const recipientForms = useRecipientForms();
```

Comment:
I think you need to pass this down to RecipientListScreenComponent, the list screen is currently still showing hard coded data

Reply:
I've left this as you were working on this page.

Reply:
makes sense